kitsune/users/models.py

```diff
@@ -67,6 +67,8 @@ class Profile(ModelBase, SearchMixin):
     # No city validation
     city = models.CharField(max_length=255, null=True, blank=True,
                             verbose_name=_lazy(u'City'))
+    involved_from = models.DateField(null=True, blank=True,
+                            verbose_name=_lazy(u'Involved with SUMO from'))
```

Comment:
This isn't correct. The bug is about the date the user started contributing to Mozilla, not SUMO. So if they were a rep for a year before they joined SUMO they could put the date when they started being a rep.